src/fetch_purchases_suppliers.py

```diff
@@ -0,0 +1,40 @@
+import requests
+import pandas as pd
+
+
+BASE_DATA_DIR = './data/'
```

Comment:
I'd recommend using `os.path` to build the path so Python handles the `os.sep` and other compatibility issues (`/` for Linux/macOs and `\` for Windows, for example).